addons/project_todo/static/src/web/activity/activity_menu_patch.js

```diff
@@ -42,4 +42,33 @@ patch(ActivityMenu.prototype, {
             size: "md",
         });
     },
+
+    availableViews(group) {
+        if (group.name === "To-Do") {
```

Comment:
Since the group name is translated is not really a good idea to do that, we could maybe add an extra key/value in the group object to know if the group is todo. 🙂 

```suggestion
        if (group.is_todo) {
```